docs/conf.py

```diff
@@ -20,12 +21,13 @@
 # -- Project information -----------------------------------------------------
 
 project = "Bayesian Tracker (btrack) 🔬💻"
-copyright = "2022, Alan R Lowe"
 author = "Alan R Lowe"
+copyright = f"2022-{datetime.date.today().year}, {author}"
```

Comment:
do you need to add the current year? Having just the initial year is okay too, or even no year

Reply:
a range is probably fine - at least it's automated. But is 2022 the correct start year? Or is it an outdated 'current' year?

Reply:
Yeah I'd argue 2022 seems too late, but I don't know when `btrack` started

Reply:
the license file is from 2018, but may be even earlier?

Reply:
looking at the [README history](https://github.com/quantumjot/BayesianTracker/commit/ece143930813b135578766d846af81505f5ce172) the starting year should be 2017

Reply:
Okay I'll add that for now